recipes/fastentrypoints/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "fastentrypoints" %}
+{% set version = "0.12" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastentrypoints-{{ version }}.tar.gz
+  sha256: ff284f1469bd65400599807d2c6284d5b251398e6e28811f5f77fd262292410b
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - fastep=fastentrypoints:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - fastentrypoints
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ninjaaron/fast-entry_points
+  summary: Makes entry_points specified in setup.py load more quickly
+  license: BSD-1-Clause
```

Comment:
Looks like in the Git repo, the license is listed as `BSD-2-Clause`, and not `BSD-1`, so please change that. You can also add `license_family: BSD` since you will be modifying the recipe. Other than this, it looks good to me.

Reply:
Thanks @npavlovikj